server/routes/messages.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+
+const router = express.Router();
+const protect = require("../middleware/auth");
+const { createNewMessage, loadMessages, deleteMessage } = require("../controllers/message");
+
+router.route("/new").post(protect, createNewMessage);
+
+router.route("/load/:conversationId").get(protect, loadMessages);
+
+router.route("/delete/:_id").delete(protect, deleteMessage);
```

Comment:
```suggestion
router.route("/delete/messageId").delete(protect, deleteMessage);
```
Why don't we consider using plain descriptive param names to increase readability? 😄